lib/liquid-fixpoint/src/constraint.rs

```diff
@@ -47,6 +47,7 @@ pub enum Sort<T: Types> {
     Bool,
     Real,
     Str,
+    // Char, // TODO:
```

Comment:
Ok, I asked around, and fixpoint doesn't support the theory of characters. It wouldn't be too hard to implement, but since we don't have a specific use case and in the interest of getting this merged, we can encode `char`s as integers for now. Does that make sense?